librz/core/cmd/cmd.c

```diff
@@ -3332,15 +3332,25 @@ static char *do_handle_ts_unescape_arg(struct tsr2cmd_state *state, TSNode arg,
 	}
 }
 
-static RzCmdParsedArgs *parse_args(struct tsr2cmd_state *state, TSNode args, bool do_unwrap) {
+static bool is_arg_raw(const RzCmdDesc *cd, size_t i) {
+	const RzCmdDescArg *arg = rz_cmd_desc_get_arg(cd, i);
+	return arg && arg->type == RZ_CMD_ARG_TYPE_RAW;
+}
+
+static RzCmdParsedArgs *parse_args(struct tsr2cmd_state *state, TSNode args, bool do_unwrap, const RzCmdDesc *cd) {
 	if (ts_node_is_null(args)) {
 		return rz_cmd_parsed_args_newargs(0, NULL);
 	} else if (is_ts_args(args)) {
+		bool old_do_unwrap = do_unwrap;
 		uint32_t n_children = ts_node_named_child_count(args);
 		uint32_t i;
 		char **unescaped_args = RZ_NEWS0(char *, n_children);
 		for (i = 0; i < n_children; i++) {
 			TSNode arg = ts_node_named_child(args, i);
+			do_unwrap = old_do_unwrap;
```

Comment:
very minor thing, but i think it's better to define in the loop a `do_unwrap_arg` and set it to `do_unwrap` and change it in the `if` below, instead of changing the global `do_unwrap` and restoring it.

Reply:
ok 632c51f2c33eb7734f12eec6a3173ec19f75353e